### PR TITLE
fix(website): Fix graphics and adjust elevator image size

### PR DESCRIPTION
**Pull request by Patticatti**
Fix conditions graphic and adjust elevator image size

### CODE REVIEW

File: website/src/components/FeatureSection/index.tsx [hunk 1]
```diff
@@ -36,7 +36,11 @@ export default function FeatureSection({
   return (
     <section className="py-16">
       <div
-        className={`max-w-screen-xl mx-auto flex justify-between items-center ${reverse ? "flex-wrap-reverse" : "flex-wrap"}`}
+        className={`max-w-screen-xl mx-auto flex justify-between items-center ${
+          reverse
+            ? "lg:flex-row-reverse flex-col-reverse"
+            : "lg:flex-row flex-col"
+        }`}
```
Comment (jamilbk) [inline website/src/components/FeatureSection/index.tsx]: I think this is breaking the flipped section.

Screenshot 2024-08-21 at 2 10 22 PM

File: website/src/app/page.tsx [hunk 2]
```diff
@@ -147,7 +147,7 @@ export default function Page() {
         }
         image={
           <Image
-            src="/images/policy-conditions.png"
+            src="/images/policy-conditions.svg"
```
Comment (jamilbk) [inline website/src/app/page.tsx]: Unfortunately this SVG is nearly 140KB which is double the compressed webp size that NextJS achieved:

Screenshot 2024-08-21 at 2 12 59 PM

**Review by jamilbk — CHANGES_REQUESTED** [review]

File: website/src/app/page.tsx (hunk 2 above)
Comment (jamilbk) [reply website/src/app/page.tsx]: I think the SVG contains lots of base64-encoded PNG data which is the reason why. Shadows gradients and such sometimes don't translate to SVG very well.

Comment (Patticatti) [reply website/src/app/page.tsx]: I see, will reupload as PNG then

Comment (jamilbk) [inline website/src/app/page.tsx]: This link is broken now.

File: website/public/images/policy-conditions.png
Comment (jamilbk) [inline website/public/images/policy-conditions.png:1]: @Patticatti I think this will be pixelated on Retina screens, you might want to export @2x

**Review by jamilbk — CHANGES_REQUESTED** [review]

File: website/public/images/policy-conditions.png
Comment (Patticatti) [reply website/public/images/policy-conditions.png:1]: Fixed to higher res png

File: website/src/components/Analytics/index.tsx [hunk 3]
```diff
@@ -5,7 +5,7 @@ import LinkedInInsights from "./LinkedInInsights";
 export default function Analytics() {
   return (
     <>
-      <Mixpanel />
+      {/* <Mixpanel /> */}
```
Comment (jamilbk) [inline website/src/components/Analytics/index.tsx]: Was this intentional? 
```suggestion
      <Mixpanel />
```

**Review by jamilbk — CHANGES_REQUESTED** [review]
I think you may have accidentally disabled Mixpanel.

**Comment by jamilbk** [comment]
@Patticatti I think there might be a minor sizing issue between the `md` and `lg` breakpoints:

Screenshot 2024-08-21 at 4 04 49 PM

**Review by jamilbk — APPROVED** [review]
Going to approve just to get this merged and then I'll fix the issues in another PR.

**Pull request merged**